src/xenia/vfs/devices/stfs_xbox.h

```diff
@@ -10,12 +10,37 @@
 #ifndef XENIA_VFS_DEVICES_STFS_XBOX_H_
 #define XENIA_VFS_DEVICES_STFS_XBOX_H_
 
+#include "xenia/xbox.h"
 #include "xenia/base/string_util.h"
 #include "xenia/kernel/util/xex2_info.h"
 
+#if XE_PLATFORM_WIN32
+#include "xenia/base/platform_win.h"
+#define timegm _mkgmtime
+#endif
+
 namespace xe {
 namespace vfs {
 
+// Convert FAT timestamp to 100-nanosecond intervals since January 1, 1601 (UTC)
+static uint64_t decode_fat_timestamp(const uint32_t date, const uint32_t time) {
```

Comment:
I'm not entirely sure why (maybe `static` results in a new copy of the function being created in every translation unit it's used in where it actually happens to be compiled to a subroutine, possibly increasing the size of the executable), but Xenia's convention is to use `inline` instead of `static` for functions defined in headers.

Reply:
Ok. I will change it to inline